hibernate-core/src/main/java/org/hibernate/engine/spi/NamedQueryDefinition.java

```diff
@@ -133,7 +134,8 @@ public NamedQueryDefinition(
 				comment,
 				parameterTypes,
 				null,		// firstResult
-				null		// maxResults
+				null,		// maxResults,
+				null 		//passDistinctThrough
```

Comment:
seems we don't need comma for `// maxResults` and we need a space char for `//passDistinctThrough`.

Reply:
fixed :)